### PR TITLE
Add average to survey results

### PR DESCRIPTION
**Pull request by juniwbjerde**
# Description
Add the average result to questions where the options start with a number as requested by Fagkom.

PS: I am aware that just adding the average based on wheter the first part of the question text is a number is a kind of hacky way to do this. I considered adding a field in the question which mapped the answers to a number and then show average when this field was set or something like that, but I imagine it would just cause more bugs as that opens up for the user to add wrong mapping etc. So yes, this is not the cleanest way to do it, but i figured it would be the most idiot-proof way.

# Result
The average shows up right under the grapgh for the question results. The number is in a pill with a color indicating the size of the number.

Before:
image

After:
image

The pill colors are in this range:
image
image
image
image

# Testing

- [ x] I have thoroughly tested my changes.

---

Resolves ABA-303

### CODE REVIEW

File: app/routes/surveys/components/Submissions/Results.tsx [hunk 1]
```diff
@@ -166,6 +175,11 @@ const Results = ({
                           dataKey="selections"
                         />
                       )}
+                      <div>
+                      {questionIsNumberic &&
+                        <p>Gjennomsnitt: {getAverage(graphData[question.id])}</p>
+                      }
+                      </div>
```
Comment (ivarnakken) [inline app/routes/surveys/components/Submissions/Results.tsx:182]: Is the `<div>` necessary?

**Review by ivarnakken — APPROVED** [review]
Kind of hacky, yes, but this is still a nice MVP 🚀 
Might consider adding an info tooltip to clarify that the number may be wrong, idk.

**Review by PeterJFB — APPROVED** [review]
Nice and simple. Couple of small type fixes + linting and it should be good 👍

File: app/routes/surveys/components/Submissions/Results.tsx [hunk 2]
```diff
@@ -138,6 +146,7 @@ const Results = ({
           const graphType = graphOptions.find(
             (a) => a.value === question.displayType
           );
+          const questionIsNumberic = question.options.reduce((result, option) => result && /^\d+/.test(option.optionText), true);
```
Comment (PeterJFB) [inline app/routes/surveys/components/Submissions/Results.tsx:149]: typo: numberic => numeric or number :)

File: app/routes/surveys/components/Submissions/Results.tsx (hunk 1 above)
Comment (juniwbjerde) [reply app/routes/surveys/components/Submissions/Results.tsx:182]: Nope, absolutely not. Will remove!

File: app/routes/surveys/components/Submissions/Results.tsx (hunk 2 above)
Comment (juniwbjerde) [reply app/routes/surveys/components/Submissions/Results.tsx:149]: Ah good catch!

**Review by ivarnakken — APPROVED** [review]
Woah very cool with the colors! Makes it soo much easier to scroll through the results quickly! 😍

It's kind of difficult to read the number on the yellow pill on dark mode, but I think that can be iterated on later, as we're probably want to introduce more yellow shades anyways.

You just need to lint this and we're good to go 🚀

**Pull request merged**